.github/workflows/aggregate.yaml

```
@@ -4,7 +4,7 @@ on:
   schedule:
     # at 4AM, 10AM, 4PM, and  10PM UTC
     # at 12AM, 6AM, 12PM, and  6PM EST
-    - cron '0 4/6 * * *'
+    - cron: '0 4/6 * * *'
```

Given that the aggregation script ignores the latest 50 transactions (to avoid chain forks), we might want to run the script 100 minutes later (50 blocks * 2 minutes p/each) or more

---

100 minutes is a little more than 1h and a half. So maybe we are OK with 2hs later
```suggestion
    - cron: '0 6/6 * * *'
```

---

fb58a063cb32e395acc1643cecc68c10b446c098